build.py

```diff
@@ -44,16 +50,18 @@ def build_mac(release):
         return False
     return True
 
+
 def check_pkgconfig(lib, min_version):
     try:
         subprocess.check_output(
             ['pkg-config', '--atleast-version', min_version, lib])
     except subprocess.CalledProcessError:
-        print("%s >= %s is missing" % (lib, min_version))
+        print(("%s >= %s is missing" % (lib, min_version)))
```

Comment:
this change is unnecessary, the statement works without the added parentheses in both python 2 and 3